.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_CITY_KEY=pk.933e2c092a691bf62d40a4264c583238
+
```

Comment:
Hi there, good job on lab so far!

Feedback: Remove .env and get it .gitignored. You will need to add `.env` to `.gitignore`. Please make sure to grab a TA if you need assistance with this 👍